### PR TITLE
feat(console): application table placeholder

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

feat(console): application table placeholder

image

## Testing

image

- [x] create an application named `My App` on the application type selected inside the placeholder.
- [x] navigate to the application guide on the application created.

## Checklist

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    display: flex;
+    justify-content: space-between;
+    align-items: stretch;
+    max-width: 736px;
+    gap: _.unit(4);
+
+    .option {
+      flex: 1;
+      display: flex;
+      flex-direction: column;
```
Comment (simeng-li) [inline packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.module.scss:32]: Looks like the card layout does not need to a flex-box?

File: packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import * as styles from './index.module.scss';
+
+const defaultAppName = 'My App';
+
+const ApplicationsPlaceholder = () => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const navigate = useNavigate();
+  const [isCreating, setIsCreating] = useState(false);
+  const [isGetStartedModalOpen, setIsGetStartedModalOpen] = useState(false);
+  const [createdApplication, setCreatedApplication] = useState<Application>();
```
Comment (simeng-li) [inline packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.tsx:27]: Merge these two states?  technically `isGetStartedModalOpen === Boolean(createdApplication)`

File: packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    }
+  };
+
+  const closeGuideModal = () => {
+    if (!createdApplication) {
+      return;
+    }
+
+    setIsGetStartedModalOpen(false);
+    toast.success(t('applications.application_created', { name: createdApplication.name }));
```
Comment (simeng-li) [inline packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.tsx:65]: Checked with Mian, could remove this toast.  The application created status has already been informed by the GuideModal itself.

File: packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.module.scss (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.module.scss:32]: Need to set the type description content to fullfill the remaining space to make buttons alignment.
image

File: packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.tsx (hunk 3 above)
Comment (xiaoyijun) [reply packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.tsx]: Updated.

File: packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/Applications/components/ApplicationsPlaceholder/index.tsx]: Updated.

**Review by simeng-li — APPROVED** [review]
LGTM

**Pull request merged**